### PR TITLE
comparing 

### PR DESCRIPTION
**Pull request by OmotosoYusuf**

### CODE REVIEW

File: src/routes/routes.js [hunk 1]
```diff
@@ -126,6 +126,10 @@ export const initRoutes = app => {
     upvoteInvite
   );
 
+  // Report User section
+  app.get('/repoetUser', (req, res) => res.render('repoetUser', { isAuth: false }));
```
Comment (LeanKhan) [inline src/routes/routes.js]: Can you rename this to reportUser please

**Comment by LeanKhan** [comment]
Please it should be `reportUser`, so change that in your routes and in the filename. \
Thanks.

**Comment by OmotosoYusuf** [comment]
Alright, I wanted to ask intially, but I just wanted to get it done on time first

**Review by OmotosoYusuf — COMMENTED** [review]
I have made the changes

File: .gitignore [hunk 2]
```diff
@@ -7,3 +7,7 @@ coverage/
 npm-debug.log
 .idea/
 .jshintrc
+
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (shonubijerry) [inline .gitignore]: it should be just `.vscode` not the content of `.vscode/settings.json`

As you can see below, the file still appear in your push

**Review by shonubijerry — COMMENTED** [review]
kindly provide a screenshot of the implementation

**Comment by OmotosoYusuf** [comment]
reportUser
reportUserMobile

**Review by shonubijerry — APPROVED** [review]

**Pull request merged**